lib/rexml/parsers/baseparser.rb

```diff
@@ -504,10 +510,9 @@ def normalize( input, entities=nil, entity_filter=nil )
 
       # Unescapes all possible entities
       def unnormalize( string, entities=nil, filter=nil )
-        rv = string.gsub( /\r\n?/, "\n" )
```

Comment:
It seems that this is a backward incompatible change.
Do we have any existing test that `string` includes `\r\n` as new line?

Reply:
There is no existing test.
I removed it because I thought it was unnecessary for the unnormalize process, but it breaks compatibility.
I will remove this change.
Sorry.